lib/src/ConnectionManager.js

```diff
@@ -3,14 +3,25 @@ import ChainWebSocket from './ChainWebSocket';
 
 class Manager {
   constructor({url, urls}) {
```

Comment:
need documentation here for developers to understand what they need to pass into this class when they instantiate it.